pom.xml

```diff
@@ -388,6 +388,16 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.junit.jupiter</groupId>
+                    <artifactId>junit-jupiter</artifactId>
```

Comment:
Brukes vi fortsatt jupiter?

Reply:
Nei, men spring-test dro inn jupiter transitivt, noe som førte til at det dukket opp i autocompleten i intellij

Reply:
Ok good to know :) så den exclusion taggen litt sent